modular_skyrat/modules/goofsec/jobs/sol_fed.dm

```diff
@@ -33,51 +37,77 @@ GLOBAL_LIST_INIT(emergency_responders, list())
 	..()
 
 /datum/antagonist/ert/request_911/greet()
-	var/missiondesc =  "<span class='warningplain'><B><font size=6 color=red>You are the [name].</font></B>"
-	missiondesc += "<BR><B><font size=5 color=red>You are NOT a Nanotrasen Employee. You work for the Sol Federation as a [role].</font></B>"
+	var/missiondesc =  ""
+	missiondesc += "<B><font size=5 color=red>You are NOT a Nanotrasen Employee. You work for the Sol Federation as a [role].</font></B>"
 	missiondesc += "<BR>You are responding to emergency calls from the station for immediate SolFed [department] assistance!\n"
+	missiondesc += "<BR>Use the Cell Phone in your backpack to confer with fellow first responders!\n"
+	missiondesc += "<BR><B>911 Transcript is as follows</B>:"
+	missiondesc += "<BR> [GLOB.call_911_msg]"
 	missiondesc += "<BR><B>Your Mission</B>:"
-	missiondesc += "<BR> <B>1.</B> Contact whoever called 911 and assist in resolving the matter."
+	missiondesc += "<BR> <B>1.</B> Contact [GLOB.caller_of_911] and assist them in resolving the matter."
 	missiondesc += "<BR> <B>2.</B> Protect, ensure, and uphold the rights of Sol Federation citizens on board [station_name()]."
 	missiondesc += "<BR> <B>3.</B> If the 911 call was made in error, for poor reasons, or with known malice, use the False 911 Call Reporter in \
-	your backpack to vote that the 911 call was false. This will immediately fine the crew and issue a response to arrest the caller."
+	your backpack to vote that the 911 call was false. This will immediately fine the crew and issue a SWAT Response to arrest those responsible. \
+	Coordinate with this SWAT team to ensure they arrest the correct people."
+	missiondesc += "<BR> <B>4.</B> When you have finished with your work on the station, use the Beamout Tool in your backpack to beam out yourself \
+	along with anyone you are pulling."
 	to_chat(owner,missiondesc)
 	var/policy = get_policy(ROLE_FAMILIES)
 	if(policy)
 		to_chat(owner, policy)
 	var/mob/living/M = owner.current
 	M.playsound_local(M, 'sound/effects/families_police.ogg', 100, FALSE, pressure_affected = FALSE, use_reverb = FALSE)
 
-/datum/outfit/ert/request_911
+/datum/outfit/request_911
 	name = "911 Response: Base"
 	back = /obj/item/storage/backpack/duffelbag/cops
 	backpack_contents = list(/obj/item/false_911_call_reporter = 1)
 
 	id_trim = /datum/id_trim/space_police
 
+/datum/outfit/request_911/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(visualsOnly)
+		return
+
+	var/obj/item/card/id/C = H.wear_id
+	if(istype(C))
+		shuffle_inplace(C.access) // Shuffle access list to make NTNet passkeys less predictable
+		C.registered_name = H.real_name
+		if(H.age)
+			C.registered_age = H.age
+		C.update_label()
+		C.update_icon()
+		var/datum/bank_account/B = SSeconomy.bank_accounts_by_id["[H.account_id]"]
```

Comment:
Why do they have a bank acount? Is this just some copy-paste of code that's already somewhere else? Why is there so many single-letter vars?

Reply:
Admittedly stole it from job code, I can drop the bank account bits.

Reply:
Would be preferrable